src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -3121,10 +3124,10 @@ namespace GraphQL.Validation.Rules
         public KnownArgumentNames() { }
         public System.Threading.Tasks.Task<GraphQL.Validation.INodeVisitor> ValidateAsync(GraphQL.Validation.ValidationContext context) { }
     }
-    public class KnownDirectives : GraphQL.Validation.IValidationRule
+    public class KnownDirectivesInAllowedLocations : GraphQL.Validation.IValidationRule
```

Comment:
But we probably should mention the fact that this class has been renamed.  Anyone that defines a custom set of validation rules will need to change that.  It can just go within the API cleanup list, probably.